Description: Tweak upper bounds of versions to not exclude minor extensions.
 haskell-hsmtlib (0.2.0.6-1) unstable; urgency=low
 .
   * Initial release.
Author: Kristoffer H Rose <krisrose@debian.org>

---
--- haskell-hsmtlib-0.2.0.6.orig/Hsmtlib.cabal
+++ haskell-hsmtlib-0.2.0.6/Hsmtlib.cabal
@@ -66,7 +66,7 @@ library
   -- other-extensions:    
   
   -- Other library packages from which modules are imported.
-  build-depends:       base >=4.6 && <= 4.7, smtLib >=1.0 && <1.1, containers >=0.5 && <0.6, pretty >=1.1 && <1.2, process >=1.1 && <= 1.2 , transformers >=0.3 && <0.4, parsec >=3.1 && <3.2
+  build-depends:       base >=4.6 && < 4.8, smtLib >=1.0 && <1.1, containers >=0.5 && <0.6, pretty >=1.1 && <1.2, process >=1.1 && < 1.3 , transformers >=0.3 && <0.4, parsec >=3.1 && <3.2
   
   -- Directories containing source files.
   -- hs-source-dirs:      
